pages/domains/domains_eligibility.md

```diff
@@ -51,13 +51,13 @@ Read more about authorizing officials for:
 - [Executive branch federal agencies](#authorizing-officials-for-executive-branch-federal-agencies)
 - [Judicial branch federal agencies](#authorizing-officials-for-judicial-branch-federal-agencies)
 - [Legislative branch federal agencies](#authorizing-officials-for-Legislative-branch-federal-agencies)
-- [U.S. Senate](#u.s.-senate)
-- [U.S. House of Representatives](#u.s.-house-of-representatives)
-- [Other legislative branch agencies](#other-legislative-branch-agencies)
+    - [U.S. Senate](#u.s.-senate)
+    - [U.S. House of Representatives](#u.s.-house-of-representatives)
+    - [Other legislative branch agencies](#other-legislative-branch-agencies)
 - [Interstate organizations](#authorizing-officials-for-interstate-organizations)
 - [U.S. states and territories](#authorizing-officials-for-u.s.-states-and-territories)
-- [States and territories: executive branch](#states-and-territories:-executive-branch)
-- [States and territories: judicial and legislative branches](#states-and-territories:-judicial-and-legislative-branches)
+    - [States and territories: executive branch](#states-and-territories:-executive-branch)
+    - [States and territories: judicial and legislative branches](#states-and-territories:-judicial-and-legislative-branches)
```

Comment:
Same for this one 
`#states-and-territories%3A-judicial-and-legislative-branches`